src/utils/resolveModuleDefault.js

```diff
@@ -1,5 +1,3 @@
-/* eslint-disable no-underscore-dangle */
-const resolveModuleDefault = module =>
-  module.__esModule ? module.default : module
+const resolveModuleDefault = module => module.default || module
```

Comment:
What is the real problem - `.default` for non ES module, or `not .default` for ES module?

Reply:
>What is the real problem - .default for non ES module, or not .default for ES module?

`.default` for non ES module.

The problem is inconsistency between dynamic imports of webpack 4 vs. babel-plugin-dynamic-import-node. This is a known issue and currently there are 2 workarounds. Both of them are described here: https://github.com/airbnb/babel-plugin-dynamic-import-node/issues/47. 

This PR replaces one of these workarounds (deleted code) with the other one (added code). As a result, `loadable-components` will have better support for webpack 4.

Reply:
React-loadable and universal-component are both looking for `_esModule`
Async and Imported component are looking only for `.default`

@neoziro - it's your turn.